onward/app/feed/DeeplyReadAgent.scala

```diff
@@ -0,0 +1,107 @@
+package feed
+
+import contentapi.ContentApiClient
+import com.gu.contentapi.client.model.v1.{Content}
+import services.{OphanApi, OphanDeeplyReadItem}
+import play.api.libs.json._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/*
+  The class DeeplyReadItem is the one that define the answer to the deeply-read.json
+  Note that it's different from OphanDeeplyReadItem which is the one we read from the Ophan Api
+
+ */
+case class DeeplyReadItem(
+    path: String,
+    benchmarkedAttentionTime: Int,
+    url: String,
+    linkText: Option[String],
+    showByline: Boolean,
+    byline: Option[String],
+    thumbnail: Option[String],
+    isLiveBlog: Boolean,
+    pillar: Option[String],
+    designType: String,
+    webPublicationDate: String,
+    headline: Option[String],
+    shortUrl: Option[String],
+)
+object DeeplyReadItem {
+  implicit val jsonWrites = Json.writes[DeeplyReadItem]
+}
+
+class DeeplyReadAgent(contentApiClient: ContentApiClient, ophanApi: OphanApi) {
+
+  /*
+      This (DeeplyReadAgent) agent is similar in purpose and interface as the ones we already have at the
+      time those lines re written, namely MostPopularAgent and its siblings, but there is a difference: we use a
+      mutable map instead of a com.gu.Box. In either case they ( mutable map and Boxes ) are essentially used as
```

Comment:
I'm not sure this PR explains anywhere _why_ it isn't using Box and sticking with the established convention.